Makefile

```diff
@@ -0,0 +1,12 @@
+
+ver:
+	@read -p "Enter a new version:" version; \
+	old_version=$(shell grep -o "\d.\d.\d" Dockerfile);  \
+	echo "$$old_version to $$version"; \
+	for i in $(shell find . -name "Dockerfile"); do sed -i "s/$$old_version/$$version/g" $$i; done;
```

Comment:
Not sure if this is helpful, but Dockerfiles can contain arguments that are passed into the `docker build` command. That might be a bit more elegent than running `sed -i` on the file.

Reply:
Cool, Github Action calls docker build in the background and developers don't have the control to modify the command.